### PR TITLE
Fix broken "Function Namespace Managers" links

### PR DESCRIPTION
**Pull request by devozerov**
```
== NO RELEASE NOTE ==
```

### CODE REVIEW

**Review by agrawalreetika — APPROVED** [review]
LGTM

**Comment by agrawalreetika** [comment]
@rohanpednekar this looks good to me for merge

File: presto-docs/src/main/sphinx/admin/function-namespace-managers.rst [hunk 1]
```diff
@@ -94,4 +94,4 @@ Name                                        Description
 See Also
 --------
 
-:doc:`create-function`, :doc:`alter-function`, :doc:`drop-function`, :doc:`show-functions`
+:doc:`/sql/create-function`, :doc:`/sql/alter-function`, :doc:`/sql/drop-function`, :doc:`/sql/show-functions`
```
Comment (steveburnett) [inline presto-docs/src/main/sphinx/admin/function-namespace-managers.rst:97]:
```suggestion
:doc:`../sql/create-function`, :doc:`../sql/alter-function`, :doc:`../sql/drop-function`, :doc:`../sql/show-functions`
```

**Review by steveburnett — CHANGES_REQUESTED** [review]
Thanks for the docs! 

1) 
Local doc build fails with
"Running Sphinx v4.4.0
Sphinx version error: The sphinxcontrib.applehelp extension used by this project needs at least Sphinx v5.0; it therefore cannot be built with this version. make: *** [html] Error 2"
Related to PR #21708.
Ran git merge master, then the doc build ran successfully.

2. 
Reviewed local doc build, and fixed the links. Update the PR and re-request my review.

**Comment by steveburnett** [comment]
Closing due to lack of activity and age. If this is still relevant, please re-open this PR, rebase, and resolve any file conflicts.

**Pull request closed without merging**